doc/news/changes/minor/20220426Jean-PaulPelteret

```diff
@@ -0,0 +1,6 @@
+Improved: The step-44 tutorial now uses FE_DGP elements to discretise the
```

Comment:
I am not sure if we strive for American English consistently, but let me suggest it anyway :wink:
```suggestion
Improved: The step-44 tutorial now uses FE_DGP elements to discretize the
```

Reply:
Thanks, I've now fixed this.